packages/@ama-sdk/create/README.md

```diff
@@ -37,11 +37,16 @@ npm create @ama-sdk typescript <project-name> -- --package-manager=yarn [...opti
 
 ## Options list
 
-- `--spec-path`: Path to the swagger/open-api specification used to generate the SDK
 - `--package-manager`: Node package manager to be used (`npm` and `yarn` are available).
 - `--debug --no-dry-run`: Enable schematics debug mode (dry-run is not currently supported).
 - `--o3r-metrics`: Enable or disable the collection of anonymous data for Otter
 - `--exact-o3r-version` : use a pinned version for [otter packages](https://github.com/AmadeusITGroup/otter/blob/main/docs/README.md).
 
+- `--spec-path`: Path to the swagger/open-api specification used to generate the SDK
+- `--spec-package-name`: The npm package name where the spec file can be fetched
+- `--spec-package-registry`: The npm registry where the spec file can be fetched
```

Comment:
I am not sure if I understood correctly but I think it would be interesting to add  a note at the end of the list of args where to explain that a token is needed to be set at user level for the specs package (if this is private). Maybe a link to the doc where Corinne explains how to handle the tokens, if this doc is public.

Reply:
you got it right, I added a note to explain that this option requires a global setup for the token